circleci/report-card

```diff
@@ -21,11 +21,22 @@ DOCKER_PASS=$2
 DOCKER_EMAIL=$3
 GITHUB_TOKEN=$4
 
+
+REPORT_CARD_IMAGE="1a90b84"
+
 echo "Logging into DockerHub..."
 docker login -u $DOCKER_USER -p $DOCKER_PASS --email="$DOCKER_EMAIL"
 
-echo "Running report-card..."
-docker run -t \
-    -v `pwd`:/repo/$(basename `pwd`) \
-    -e GITHUB_API_TOKEN=$GITHUB_TOKEN \
-    clever/report-card --repo /repo/$(basename `pwd`) --publish
+echo "Pulling report-card:$REPORT_CARD_IMAGE ..."
+timeout 5m docker pull clever/report-card:$REPORT_CARD_IMAGE
```

Comment:
I think we are mostly likely to hit the timeout when pulling new images (not too often). And since the image is large,  so I want to give it enough time to pull it before timing out. Locally I've seen this take a few minutes.